content/battery.md

```diff
@@ -19,167 +19,155 @@ section: hardware
+## Screen brightness
 
-## Power Profiles
+One of the largest consumers of laptop power is the display backlight. Up to 10% more battery life can be gained by reducing display brightness. Our laptops don't currently have an ambient light sensor, but brightness can be adjusted manually via the **System Menu** (see screenshot below) or with the keyboard shortcuts for each individual laptop model, which can be found on the "External Overview" pages of [each model's technical documentation](https://support.system76.com/articles/guides/).
 
-With the <u>system76-power</u> package there are **Power Profiles** that can be accessed in the **System Menu** in the image below.
+![Brightness](/images/battery/brightness.png)
 
-![Battery](/images/battery/system-menu.png)
+## Power Profiles
 
-## Useful Programs
+Pop!\_OS comes preloaded with the `system76-power` package, which allows you to select **Power Profiles** through the **System Menu**, as shown in the image below. This screenshot is from a laptop that has switchable Nvidia graphics, and a laptop that has only Intel graphics will not hawe those options.
 
-### TLP
+![Battery](/images/battery/power-menu.png)
 
-<u>TLP</u> is an excellent program for increasing battery life on all of our laptops. <u>TLP</u> is a pure command line tool with automated background tasks and does not contain a GUI. Its default settings are excellent for most situations and require little tuning. <u>TLP</u> will take care of most of the settings that <u>Powertop</u> autotuning would, and with less trial and error.
+The system will default on each startup to the "Balanced" setting, which is usually preferred even when connected to AC power as it helps keep the system running cool and quiet. When maximum performance is needed, the "High Performance" setting will uncap everything and let the system run as hot and power-hungry as it wants, which can dramaticaly reduce battery life and may also increase the noise made by the system fan(s).
 
-To install TLP, run this command:
+The "Battery Life" setting will do a number of things to improve battery life, including reduce the screen brightness (which can be turned back up as described above if desired), reduce the maximum speed of the CPU (and any GPUs if present), aggressively control how applications are allowed to use the CPU and GPU, and much more under the hood.
 
-```bash
-sudo apt install tlp tlp-rdw --no-install-recommends
-```
+## Switchable graphics settings
 
-TLP will take effect upon restart. To see current configuration settings, run this command:
+The settings for the graphics modes on laptops that have switchable Nvidia graphics are outlined in detail [on their own support document](https://support.system76.com/articles/graphics-switch-pop), but to summarize:
```

Comment:
After searching a few other articles, it appears we use NVIDIA in all-caps and we should be consistent with that here as well.

Reply:
Done, will be updated on the next git push.